### ISSUE TITLE
Autogenerated and check in examples/index.html

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria-practices#1079]
When merging into master, we should auto-generate and check `examples/index.html` into master. Can this happen in the travis deploy script?

**Comment by spectranaut** [comment]
@nschonni my impression is that this should be easy to add to the deploy, but looking at the `/scripts/travis-before_deploy.sh` and `scripts/travis-deploy.sh` I'm not sure how to fit it. Maybe another script that runs before the before. I'm also curious if committing into master will cause trigger travis to run again. Do you know how to do this easily? Otherwise I'll dig into it more!

**Comment by nschonni** [comment]
You might be able to commit with a `[skip ci]` or `[ci skip]` to avoid a recursive build

**Comment by nschonni** [comment]
Another thought would be to skip checking the generated file into master  at all, and just generate it for the gh-pages branch.

**Comment by mcking65** [comment]
One reason why I want it in master is because it then feeds our other publication branches, e.g., we merge from master into apg-1.2, and later apg-1.3. However, since we don't do that often, I guess that can be manual as I just did for the working draft prep.

If it is a much smaller and simpler ask, perhaps just generate automatically for the gh-pages branch.

**Comment by nschonni** [comment]
@mcking65 were you wanting it so you'd get a diff when merging other branches?

**Comment by zcorpan** [comment]
I would prefer one of the following:

* The examples/index.html file is not checked in at all, but is only generated for publication.
* The examples/index.html file is checked in, and there is some enforcement to regenerate it.

The other approaches discussed above seem like they could result in things getting out of sync or otherwise be hard to follow what's going on, in my opinion.

Since @mcking65 said he wanted the file checked in to master, I've implemented the enforcement option in #1365.
